packages/create-next-app/templates/default/pages/index.js

```diff
@@ -60,7 +59,19 @@ export default function Home() {
         >
           Powered by{' '}
           <span className={styles.logo}>
-            <Image src="/vercel.svg" alt="Vercel Logo" width={72} height={16} />
+            <svg
```

Comment:
I thought we want to discourage inlining SVGs as this bloats bundle size and prevents caching separately?